Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "TrixiParticles"
 uuid = "66699cd8-9c01-4e9d-a059-b96c86d16b3a"
 authors = ["erik.faulhaber <44124897+efaulhaber@users.noreply.github.com>"]
-version = "0.2.4-dev"
+version = "0.2.4"
```

Comment:
I think it makes more sense to wait for #529 before we release.

Reply:
Depends on how long that still will take...